src/cgnslib.c

```diff
@@ -361,17 +361,15 @@ int cg_open(const char *filename, int mode, int *file_number)
             }
             break;
         case CG_MODE_WRITE:
-            /* unlink is now done in cgio_open_file */
+            /* set default file type if not done */
```

Comment:
Maybe keep the original comment concerning unlink

Reply:
Fine by me.